### ISSUE TITLE
SwiftHTTPServer breaches the LICENSE of Representor

### ISSUE THREAD

**Issue by kylef** [paulofaria/HTTP-Server#1]
The code introduced in 8c40dd5b5bdf1405242aa073434daca41b6e4ef5 is adapted from the [Representor](https://github.com/the-hypermedia-project/representor-swift) which doesn't retain the original [license and copyright](https://github.com/the-hypermedia-project/representor-swift/blob/master/LICENSE).

The LICENSE requires that any substantial portions or copies of the software (Representor) must retain the original copyright notice.

> The above copyright notice and this permission notice shall be included in all
> copies or substantial portions of the Software.

Please update to include the original LICENSE and copyright or remove these portions of the code from SwiftHTTPServer.

**Comment by paulofaria** [comment]
Sorry @kylef. I was just playing with `Representor`. I wasn't trying to steal your code or anything. I was trying to adapt the concepts to a `Foundation` free constraint, which I would propose to the original `Representor` repo when I got something good. My goal is to make `Representor` better like when I proposed [multiple transitions with the same relation](https://github.com/the-hypermedia-project/representor-swift/issues/22). I'll remove the code from this repo anyway.
